lib/pbench/server/api/resources/users_api.py

```diff
@@ -349,25 +376,10 @@ def get(self, target_username):
             abort(HTTPStatus.INTERNAL_SERVER_ERROR, message="INTERNAL ERROR")
         if target_user:
             response_object = target_user.get_json()
-            return make_response(jsonify(response_object), HTTPStatus.OK)
-        elif self.auth.token_auth.current_user().is_admin():
-            self.logger.warning(
-                "User {} performed get operation but user {} is not found.",
-                self.auth.token_auth.current_user().username,
-                target_username,
-            )
-            abort(
-                HTTPStatus.NOT_FOUND, message=f"User {target_username} not found",
-            )
+            return make_response(jsonify(response_object), http_status)
```

Comment:
I wouldn't bother using `http_status` here -- it's supposed to be `OK`, right?  So, remove the obfuscation and potential pitfall.

Reply:
Ditto lines 470 and 520.

Reply:
Well if we come at this line the http_status is going to be `OK` but I can see the confusion.

Reply:
Yes, `http_status` _should_ be `OK` at this point...but, if you're reading the code you have to go kind of a long way to prove that to yourself.  And, since `OK` is what we _want_ to return here, there's no reason to create an opportunity for something else to happen by using a variable instead of a "constant".